### PR TITLE
Prometheus 2.0

### PR DESCRIPTION
**Pull request by d-romualdi**

### CODE REVIEW

File: src/package.xml [hunk 1]
```diff
@@ -1,363 +1,312 @@
...
-			<item name="OutputOptionSpatial" displayName="Prometheus" showIf="inPipeline"/>
-	  </layout>
-	</layouts>
-	<updateProvider 
-		schemaVersion="2" 
-		minPackageVersion="0.4.11" 
-		className="SyncroSim.Core.XMLUpdateProvider" 
-		classAssembly="SyncroSim.Core"/>
-</package>
+<package name="burnP3PlusPrometheus" displayName= " BurnP3+ (Prometheus)" description= "Prometheus fire growth model for BurnP3+" version="2.0.0" url="https://github.com/burnp3/burnp3plusprometheus">
```
Comment (katieb1) [inline src/package.xml]: Can you change displayName to "BurnP3+Prometheus"?

File: src/package.xml [hunk 2]
```diff
@@ -1,363 +1,312 @@
...
+
+	<builtFor package = "burnP3Plus" version = "2.0.0"/>
+
+	<!--Library Datasheets-->
+
+	<dataSheet name="PrometheusMultithreading" displayName="Prometheus Multithreading" isSingleRow="True" dataScope="Library">
+		<column name="EnableMultithreading" displayName="Enable Multithreading" dataType="Boolean"/>
```
Comment (katieb1) [inline src/package.xml]: Remove column "EnableMultithreading" (as discussed w/ Leonardo)

File: src/package.xml [hunk 3]
```diff
@@ -1,363 +1,312 @@
...
-		className="SyncroSim.Core.XMLUpdateProvider" 
-		classAssembly="SyncroSim.Core"/>
-</package>
+<package name="burnP3PlusPrometheus" displayName= " BurnP3+ (Prometheus)" description= "Prometheus fire growth model for BurnP3+" version="2.0.0" url="https://github.com/burnp3/burnp3plusprometheus">
+
+	<builtFor package = "burnP3Plus" version = "2.0.0"/>
+
+	<!--Library Datasheets-->
+
+	<dataSheet name="PrometheusMultithreading" displayName="Prometheus Multithreading" isSingleRow="True" dataScope="Library">
```
Comment (katieb1) [inline src/package.xml]: Change dataScope to scenario (can just remove `dataScope="Library"` as default is scenario)

File: src/growth-pandora.R [hunk 4]
```diff
@@ -217,6 +222,13 @@ parameterFilePlaceHolders <- list(
 
 ## Extract relevant parameters ----
 
+# Check if multithreading enabled
+if (Multithreading$EnableMultithreading) { # default is TRUE
+  numThreads <- Multithreading$ThreadsPerIteration
```
Comment (katieb1) [inline src/growth-pandora.R]: remove if/else statement (lines 225-230) and just keep this line: 
`numThreads <- Multithreading$ThreadsPerIteration`

File: src/package.xml [hunk 5]
```diff
@@ -1,363 +1,312 @@
...
+			<item name="sfc" displayName="Surface Fuel Consumption" dataSheet="OutputSurfaceFuelConsumptionMap" column="FileName"/>
+			<item name="cfb" displayName="Crown Fraction Burned" dataSheet="OutputCrownFractionBurnedMap" column="FileName"/>
+			<item name="cfc" displayName="Crown Fraction Consumed" dataSheet="OutputCrownFractionConsumedMap" column="FileName"/>
+			<item name="tfc" displayName="Total Fuel Consumption" dataSheet="OutputTotalFuelConsumptionMap" column="FileName"/>
+		</group>
+	</layout>
+
+	<!--Library Layout-->
+	<layout type="Library">
+		<item name="PrometheusMultithreading"/>
```
Comment (katieb1) [inline src/package.xml]: Remove Library Layout section (lines 297-300) and move PrometheusMultithreading item to scenario layout.

File: src/package.xml [hunk 6]
```diff
@@ -1,363 +1,312 @@
...
+	</layout>
+
+	<!--Project Layout-->
+	<layout type="Project">
+		<item name="FuelCodeCrosswalk" displayName="Prometheus Crosswalk"/>
+	</layout>
+
+	<!--Scenario Layout-->
+	<layout type="Scenario">
+		<item name="OutputOptionSpatial" displayName="Prometheus" showIf="inPipeline"/>
```
Comment (katieb1) [inline src/package.xml:303]: Add PrometheusMultithreading datasheet to scenario layout
`<item name="PrometheusMultithreading"/>`

**Review by katieb1 — CHANGES_REQUESTED** [review]

File: src/package.xml (hunk 1 above)
Comment (katieb1) [reply src/package.xml]: Just remove the space from the beginning of the display name and it should be good!
" BurnP3+ (Prometheus)" --> "BurnP3+ (Prometheus)"

File: src/package.xml (hunk 3 above)
Comment (d-romualdi) [reply src/package.xml]: Should we move this to the Scenario Datasheets section?

**Review by katieb1 — APPROVED** [review]
looks good!

**Pull request merged**